ocs_ci/ocs/constants.py

```diff
@@ -222,6 +223,7 @@
 TEST_FILES_BUCKET = "ocsci-test-files"
 ROOK_REPOSITORY = "https://github.com/rook/rook.git"
 OPENSHIFT_STORAGE_NAMESPACE = "openshift-storage"
+OPENSHIFT_STORAGE_EXTENDED_NAMESPACE = "openshift-storage-extended"
```

Comment:
Can this one be something else than this one? Like we can have custom namespace?

If so, I would not rely on this one value from constants but rather have it defined in config - in past we had several issues because of using namespace from constants, mainly for managed cluster deployments - provider/client and so on, where we have different namespaces as well - in this case it should be configurable based on config which I prefer here as well in case it's possible to have custom namespace name for external cluster.